Source/Charts/Data/Implementations/ChartBaseDataSet.swift

```diff
@@ -403,7 +403,7 @@ open class ChartBaseDataSet: NSObject, IChartDataSet, NSCopying
     open override var debugDescription: String
     {
         return (0..<entryCount).reduce(description + ":") {
-            $0 + "\n" + (self.entryForIndex($1)?.description ?? "")
+            "\($0)\n\(self.entryForIndex($1)?.description ?? "")"
```

Comment:
For readability
```
"""
\($0)
\(self.entryForIndex($1)?.description ?? "")
"""
```

Reply:
@jjatie we are good to merge? if this is the case, I can modify and proceed